cmd/crowdsec-cli/explain.go

```diff
@@ -85,11 +115,11 @@ cscli explain --dsn "file://myfile.log" --type nginx
 				log.Fatalf("fail to run crowdsec for test: %v", err)
 			}
 
-			// rm the temporary log file if only a log line was provided
-			if logLine != "" {
+			// rm the temporary log file if only a log line/stdin was provided
+			if tmpFile != "" {
 				f.Close()
```

Comment:
Hey! thanks @buixor ! just wondering if we need to close the file handler twice?

Reply:
yes i think the file is already close before